lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -335,6 +335,7 @@ def response_options(result)
       end
 
       def avs_code_from(transaction)
+        transaction.avs_error_response_code ||
         avs_mapping["street: #{transaction.avs_street_address_response_code}, zip: #{transaction.avs_postal_code_response_code}"]
```

Comment:
Can you indent this so it's more obvious that it's logically linked to the line above?